src/sst/core/exit.cc

```diff
@@ -12,6 +12,7 @@
 #include "sst_config.h"
 
 #include "sst/core/exit.h"
+#include <exception>
```

Comment:
Per the coding standards, includes of standard headers should go after all sst includes.  This is probably contributing to the clang-format failure.

Reply:
I ran `clang-format` but my version is newer than v12 and I didn't feel like trying to go back to an older version.

Reply:
That's not promising for moving clang-format forward if it doesn't follow the header ordering settings in the .clang-format file.  You can look at the diff of what it got with what it wanted in the "checks" tab in the PR.  Just select the clang-format check and click in the Matrix box to bring up the results of the test.

Reply:
It's too late to fix it now, but `clang-format` should have a `version 1.2.3` directive for `.clang-format` files so that it behaves like a certain version. But as the number of versions increases, this becomes hard to test, and complicates the code to behave differently for different versions, although if this versioning scheme is done from the beginning, the versioned tests can be developed incrementally.